httpServer/ui/components/dashboard.js

```diff
@@ -362,9 +391,31 @@ export default class Dashboard extends React.Component {
                             <SaveIcon size="md" />
                         </button>
                         <button className={'btn ' + (this.state.settings.ACTIVE === 0 ? 'inactive' : 'running')}
-                            onClick={() => this.saveSetting('ACTIVE', this.state.settings.ACTIVE === 0 ? 1 : 0)}>
+                            onClick={() => this.closeButton_OnClick()}>
                             <OnOffIcon size="md" />
                         </button>
+                        <Dialog
+                            open={this.state.showCloseDialog}
```

Comment:
The first attribute can be best left on the same line as the tag to stay consistent with the rest of the code

Reply:
Done as requested